examples/automm/object_detection/quick_start_on_a_tiny_dataset.py

```diff
@@ -44,27 +56,44 @@ def tutorial_script_for_quick_start():
     print("The finetuning takes %.2f seconds." % (train_end - start))
 
     # Evaluate
-    predictor.evaluate(test_path)
+    #print("Predict...")
+    #predictor.predict(test_path)
+    
+    #sample_image_path = "/home/ubuntu/ag/autogluon/examples/automm/object_detection/tiny_motorbike_coco/tiny_motorbike/JPEGImages/000038.jpg"
+    #predictor.predict([sample_image_path]*10)
+    #exit()
+    
+    print("Predict no label...")
+    predictor.predict(test_path_no_label)
 
-    eval_end = time.time()
-    print("The evaluation takes %.2f seconds." % (eval_end - train_end))
+    predictor.evaluate(test_path)
+    
+    #eval_end = time.time()
+    #print("The evaluation takes %.2f seconds." % (eval_end - train_end))
 
     # Load and reset num_gpus
-    new_predictor = MultiModalPredictor.load("./quick_start_tutorial_temp_save")
-    new_predictor.set_num_gpus(1)
-
-    # Evaluate new predictor
-    new_predictor.evaluate(test_path)
-
-    # Load Trained Predictor from S3
-    zip_file = "https://automl-mm-bench.s3.amazonaws.com/object_detection/quick_start/AP50_433.zip"
-    download_dir = "./AP50_433"
-    load_zip.unzip(zip_file, unzip_dir=download_dir)
-    better_predictor = MultiModalPredictor.load("./AP50_433/quick_start_tutorial_temp_save")
-    better_predictor.set_num_gpus(1)
+    #new_predictor = MultiModalPredictor.load("./quick_start_tutorial_temp_save")
+    #new_predictor.set_num_gpus(1)
 
     # Evaluate new predictor
-    better_predictor.evaluate(test_path)
+    #print("Predict with loaded predictor...")
+    #new_predictor.predict(test_path)
+    #print("Predict no label with loaded predictor...")
+    #new_predictor.predict(test_path_no_label)
```

Comment:
should we remove the dead code?

Reply:
Currently we do not support multiple predicts in DDP strategy. Will uncomment them in next PR.